code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -685,18 +688,22 @@
 				O.show_message(text("<span class='notice'>[]'s eyes blink and become clearer.</span>", M), 1) // So observers know it worked.
 		// Vamps react to this like acid
```

Comment:
Breaks encapsulation, but it's hopelessly broken for our med/chem systems anyway.